lingpy/basic/parser.py

```diff
@@ -364,7 +365,9 @@ def _apply(key, s, *args, **kwargs):
             try:
                 res = function(s, *args, **kwargs)
             except:
-                raise ValueError('Could not convert item "{0}" (ID: {1}).'.format(s, key))
+                print(s, key)
```

Comment:
Kind of. This info was previously included in the message passed into `ValueError`; but non-ASCII error messages made nose choke (on py2.7).